src/xpu/sycl/types.hpp

```diff
@@ -30,6 +30,21 @@ namespace impl {
 namespace xpu {
 namespace sycl {
 
+#define DNNL_ARG_SRC_4 5
```

Comment:
It doesn't look like this code belongs here. Can you move it to `sum_kernels.hpp`?

Reply:
done